src/ui/components/TitleBar.tsx

```diff
@@ -169,7 +169,7 @@ const TitleBar = () => {
 	}
 
 	return (
-		<nav className="navbar navbar-border navbar-light title-bar">
+		<aside className="navbar navbar-border navbar-light title-bar">
```

Comment:
Would it make more sense for this to be `header` rather than `aside`?

Reply:
The HTML5 `<header>` element is the equivalent of adding `role="banner"` to a generic element. Per WCAG there can only be one such element per page and it needs to contain very specific things like the site logo and navigational elements. That's why I placed the role of "banner" on the element on the top of the page. Since I couldn't re-use it here but WCAG recommends all content be in a landmark I used this more general role here. This allows AT users to get oriented around the page and to jump to different parts using keyboard shortcuts. 

The only other option would have been to move this container with the `<main>` in which case it would already have been inside a landmark and not need its own.